sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -98,6 +110,23 @@ public class CosmosTemplateIT {
 
     private static final String WRONG_ETAG = "WRONG_ETAG";
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final JsonNode NEW_PASSPORT_IDS_BY_COUNTRY_JSON = OBJECT_MAPPER.convertValue(NEW_PASSPORT_IDS_BY_COUNTRY, JsonNode.class);
+
+    private static final CosmosPatchOperations operations = CosmosPatchOperations
+        .create()
+        .replace("/age", PATCH_AGE_1);
+
+    CosmosPatchOperations multiPatchOperations = CosmosPatchOperations
+        .create()
+        .replace("/firstName", PATCH_FIRST_NAME)
```

Comment:
should we also test `set` and `remove` patch operations

Reply:
Yes we should, @TheovanKraay please add these tests.

Reply:
Added the two extra operations in multiPatchOperations.